python/src/aiconfig/editor/client/src/shared/prompt_schemas/OpenAIVisionParserPromptSchema.ts

```diff
@@ -0,0 +1,124 @@
+import { PromptSchema } from "../../utils/promptUtils";
+
+export const OpenAIVisionParserPromptSchema: PromptSchema = {
+  input: {
+    type: "object",
+    required: ["data"],
+    properties: {
+      data: {
+        type: "string",
+      },
+      attachments: {
+        type: "array",
+        items: {
+          type: "attachment",
+          required: ["data"],
+          mime_types: ["image/*"],
+          properties: {
+            data: {
+              type: "string",
+            },
+          },
+        },
+      },
+    },
+  },
+  model_settings: {
+    type: "object",
+    properties: {
+      model: {
+        type: "string",
+      },
+      system_prompt: {
+        type: "string",
+      },
```

Comment:
For future, wondering if there's a better way of refactoring this so that you can compose different schemas together. It seems like this is used in a bunch of them (if not all?) and having to repeat seems annoying

Reply:
Yes, definitely. When we move to python side we can have reusable constants to construct the schemas from